gulpfile.js

```diff
@@ -10,11 +14,34 @@ gulp.task("build:backend:assets:copy-js-vendors", function() {
       .pipe(gulp.dest("./js/vendors"));
 });
 
+gulp.task("build:backend:sass:generate-css", function() {
+  return gulp.src([
+    "./src/Backend/Core/Layout/Sass/screen.scss",
+    "./src/Backend/Core/Layout/Sass/debug.scss",
+  ])
+      .pipe(sourcemaps.init())
+      .pipe(sass({
+        includePaths: [
+          "./node_modules/"
```

Comment:
Maybe you can also add the path for the bootstrap sass files, so you only need to import the file name in the _bootstrap-imports.scss file, the file will be clearer it think